airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -167,7 +193,11 @@ def path(self, **kwargs) -> str:
 
     def stream_slices(self, stream_state: Mapping[str, Any] = None, **kwargs) -> Iterable[Optional[Mapping[str, any]]]:
         stream_state = stream_state or {}
-        start_date = pendulum.from_timestamp(stream_state.get("start_ts", self._default_start_date.timestamp()))
+        try:
```

Comment:
what problem is this `try/except` solving? (should be a comment)
can we do without it?

Reply:
deleted this block